clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+type simpleTestExecutor struct{}
```

Comment:
Should be called fakeTestExecutor (not simple)

@shyamjvs @krzysied

Reply:
I don't think so. IIUC this isn't a fake implementation, it was just left to be implemented later.
Anyway, this has been changed now I think.

Reply:
Krzysiek to confirm.

Reply:
In current version most of the test execution logic is implemented.
A lot of TODOs missing here, so it can be misleading.